cli/README.md

```diff
@@ -0,0 +1,27 @@
+# GitHub Action for the Ballerina CLI
+
+The GitHub Action for [Ballerina CLI](https://ballerina.io/) wraps the `ballerina` command inside of an Action.
```

Comment:
```suggestion
The GitHub Action for the [Ballerina CLI](https://ballerina.io/) wraps the `ballerina` command inside an Action.
```

Reply:
Thank you for the suggestion, I based the wording based on https://github.com/Azure/github-actions/tree/master/cli. Your one is better, and is more grammatically correct I guess.